frontend/package.json

```diff
@@ -14,8 +14,10 @@
         "jspdf-autotable": "^3.5.14",
         "prop-types": "^15.7.2",
         "react": "^16.13.1",
+        "react-alice-carousel": "^2.5.1",
         "react-apexcharts": "^1.3.9",
         "react-burger-menu": "^2.6.13",
+        "react-collapsible": "^2.8.3",
```

Comment:
@manifau @IremToroslu please add the new additions to the BOM as well!

https://docs.google.com/spreadsheets/d/1riRj26jKjfG1dEwkKn4bxvcrr5PahKq9wyz4ao3RDgI/edit#gid=927854276